relayer/contracts/beefy_client.go

```diff
@@ -72,12 +72,13 @@ type BeefyClientValidatorSet struct {
 
 // Uint16ArrayArray is an auto generated low-level Go binding around an user-defined struct.
 type Uint16ArrayArray struct {
-	Data []*big.Int
+	Data   []*big.Int
+	Length *big.Int
```

Comment:
Unrelated to this PR but I think this comes from `./scripts/init.sh` or `start-services.sh`

Reply:
Yeah this was that change I did to add length to UInt16Array in the solidity and start-services.sh rebuilt the contracts.